scrapy/Fetch/__init__.py

```diff
@@ -0,0 +1,43 @@
+from scrapy.core import config
+from scrapy.core import engine
+
+from scrapy.utils.log import logformatter_adapter, failure_to_exc_info
+from scrapy import signals
+from scrapy.core.downloader import Downloader 
+from twisted.internet.defer import Deferred
+import asyncio
+import logging
+import scrapy
+import warnings
```

Comment:
See https://www.python.org/dev/peps/pep-0008/#imports for import standard rules.

Specially ordering:

    1. Standard library imports.
    2. Related third party imports.
    3. Local application/library specific imports.